CondFormats/HGCalObjects/interface/HGCalMappingModuleIndexer.h

```diff
@@ -156,6 +158,10 @@ class HGCalMappingModuleIndexer {
         uint32_t internalData_offset = globalTypesNWords_[type_val] * typeCounters[type_val];
         fedit.chDataOffsets_[i] = baseData_offset + internalData_offset;
 
+        //enabled erx flags
+        //FIXME: assume all eRx are enabled now
+        fedit.enabledErx_[i] = (0b1 << globalTypesNErx_[type_val]) - 0b1;
+
```

Comment:
@pfs Any suggestions on how to implement this in the geometry file?

Reply:
I think this is what we discussed with @yulunmiao in the last Hackathon. There'll be an extra vector in the indexer which can be used to retrieve directly the erx flags. This extra vector will be present in the `FEDReadoutSequence_t` struct.